README.md

```diff
@@ -1,3 +1,5 @@
+> **Note**: We do not recommend using the charts in this repository directly except for development. Instead, please see https://artifacthub.io/packages/search?repo=spiffe
+
 > **Note**: All the helm charts in this repo are beta. We encourage you to try them out and contribute. The API may change as we move towards a production ready release.
 
```

Comment:
I think we're putting things in the wrong order, It's beta software, and that's why you don't use them directly.
```suggestion
**Note**: We do not recommend using the charts in this repository directly. Instead, please see https://artifacthub.io/packages/search?repo=spiffe

```